source/mainnet/docs/desktop-wallet/install-ledger-app.rst

```diff
@@ -48,378 +52,437 @@ The LEDGER device will generate the unique 24-word recovery phrase that is used
 
-#. To install Python3, Pip3, `libusb <https://libusb.info/>`_, and `libjpeg <http://libjpeg.sourceforge.net/>`_, copy the following into the Terminal and press Enter:
+         - :substitution-code:`|ledger-app-public-key|`
 
-   .. code-block:: console
+      #. Press both buttons when the LEDGER device says **Trust certificate**.
 
-      $brew install python@3.9 libusb libjpeg
+      #. Enter your PIN. The certificate has now been installed on the LEDGER device.
 
-   You can use `pyenv<https://github.com/pyenv/pyenv>` if you need multiple python versions. Installing libjpeg is only necessary if you have a Mac with an M1 or similar Apple Silicon CPU.
+   .. group-tab:: macOS
 
-#. To install ledgerblue, copy the following into the Terminal and press Enter:
+      #. Close all applications that might be connected to the LEDGER device such as LEDGER LIVE and Concordium Desktop Wallet and keep them closed until you've completed the steps in this guide.
 
-   .. code-block:: console
+      #. Disconnect the LEDGER device from your computer.
 
-      $pip3 install ledgerblue
+      #. Now you need to start recovery mode on the LEDGER device.
 
-Install the custom certificate using macOS
-------------------------------------------
+         - On the LEDGER device, press the *left* button and hold it down while you reconnect the LEDGER device to the computer. Navigate to **Recovery mode** and press both buttons to enter **recovery mode**.
 
-You now have to install a custom certificate to ensure that the LEDGER device trusts applications signed by Concordium's private key.
+      #. Enter your PIN code.
 
-#. Close all applications that might be connected to the LEDGER device such as LEDGER LIVE and Concordium Desktop Wallet and keep them closed until you’ve completed the steps in this guide.
+      #. Open the `Terminal <https://support.apple.com/en-gb/guide/terminal/apd5265185d-f365-44cb-8b09-71a064a42125/mac>`_ application.
 
-#. :ref:`Download the ZIP folder with the Concordium LEDGER application<Ledger-downloads>`.
+      #. Navigate to where you have downloaded and extracted the LEDGER install package. For example to navigate to the default download directory in a Terminal, enter
 
-#. Extract the files from the ZIP folder to a folder on your computer.
+         .. tabs::
 
-#. Disconnect the LEDGER device from your computer.
+            .. group-tab:: Nano S
 
-#. Now you need to start recovery mode on the LEDGER device.
+               .. code-block:: console
+                  :substitutions:
 
-   - On the LEDGER device, press the *left* button and hold it down while you reconnect the LEDGER device to the computer. Navigate to **Recovery mode** and press both buttons to enter **recovery mode**.
+                  $cd ~/Downloads/concordium-governance-ledger-app-|ccd-governance-version|-nanos-|nano-s-firmware|
 
-#. Enter your PIN code.
+            .. group-tab:: Nano S+
```

Comment:
This section should not be executed by `Nano S+` users, but can also be nice to keep the section in case the custom certificates are supported again on the devices.

Reply:
Good catch, will remove it for now.